feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/vif.c

```diff
@@ -430,16 +431,22 @@ static int vif_config_security_set(struct blob_buf *b,
 					auth_server = local_server;
 					acct_server = local_server;
 				}
+				auth_port = "1812";
+				auth_secret = "secret";
+				acct_port = "1813";
+				acct_secret = "secret";
```

Comment:
Code seems to make sense but is the secret really "secret"?
Doesn't this imply there is a RADIUS server somewhere with secret == "secret"?
Seems like a bad idea.

Reply:
This is the communication between hostapd which is running inside DR/BDR/OR and the proxy service running inside the DR.  The radius endpoint here is the proxy service.

Reply:
In other words, the server ip will always point to the entity running the radius proxy service. In case of DR it'll be 127.0.0.1.  In case of OR/BDR it will be the IP address of the DR.